client/app/services/url.js

```diff
@@ -0,0 +1,25 @@
+import { pick, extend } from "lodash";
+
+const link = document.createElement("a"); // the only way to get an instance of Location class
```

Comment:
If that's what we have :man_shrugging:, at least only one `<a>` object goes to the DOM.

Reply:
After your comment I realized that probably Wepack can polyfill Node's `url` module. But it will require more work - it does not "know" about `<base>` tag, so we'll need to get base href from the DOM, manually resolve relative URLs, etc. `<a>` tag already does all that things - you just assign _any_ URL to `a.href`, and browser properly resolves it according all its internal rules. It means that we will always have :100: correctly resolved URL

Reply:
> it does not "know" about <base> tag

It might not be relevant at the moment, but just wanted to note that we will probably stop relaying on `<base>` tag for the multi-org functionality.

Reply:
That will be cool :+1: But it's already implemented, and I think it's right to keep it working until something changes.

Reply:
I agree, just something to keep in mind.